src/Lykke.Service.HFT/Controllers/OrdersController.cs

```diff
@@ -47,13 +47,17 @@ public class OrdersController : Controller
         /// Get the last client orders.
         /// </summary>
         /// <param name="status">Order status</param>
-        /// <param name="take">Default 100; max 500.</param>
+        /// <param name="take">The amount of orders to take, default 100; max 500.</param>
+        /// <param name="skip">The amount of orders to skip, default 0</param>
```

Comment:
I don't think we need to allow our clients to dig through orders history. I suppose this should be like https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md
GET /api/v3/allOrders